core/transfer/local/transfer.go

```diff
@@ -54,8 +56,17 @@ func NewTransferService(cs content.Store, is images.Store, tc TransferConfig) tr
 	if tc.MaxConcurrentUploadedLayers > 0 {
 		ts.limiterU = semaphore.NewWeighted(int64(tc.MaxConcurrentUploadedLayers))
 	}
+	if tc.MaxConcurrentDownloadsPerLayer > 0 {
+		ts.limiterU = semaphore.NewWeighted(int64(tc.MaxConcurrentDownloadsPerLayer))
```

Comment:
Is this correct? why is `limiterU` value being reset here?

Reply:
good catch, this is not correct; fixing.

Reply:
typo copy paste, good catch, this is quite wrong, removing